build-in-container/scripts/setup.sh

```diff
@@ -59,6 +59,37 @@ check_specs() {
     fi
 }
 
+# enable custom-repo.repo to install RPMs from
+setup_custom_repofile() {
+    echo "------------ Setting up custom repofile ------------"
+    echo -e "\n" >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
+    cat $RPM_repo_file >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
+    echo -e "\n" >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
+}
+
+# enable custom blob storage to install RPMs from
+setup_custom_repo_storage() {
```

Comment:
As above, we need to also support `PACKAGE_URL_LIST` to support toolchain creation.